src/components/RepositoryTable.js

```diff
@@ -4,6 +4,7 @@ import PropTypes from 'prop-types';
 export default class RepositoryTable extends React.Component {
   static propTypes = {
     repositories: PropTypes.array.isRequired,
+    onSelectRepo: PropTypes.func,
```

Comment:
should be `onSelectRepository`, we try to avoid jargon and the word `repo` in that case.